monitoring/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@keep-network/tbtc-v2-monitoring",
+  "version": "1.0.0-dev",
+  "license": "MIT",
```

Comment:
Nitpicking but let's use `GPL-3.0-only` for all the new stuff, this is aligned with [tbtc-v2 code](https://github.com/keep-network/tbtc-v2/blob/main/solidity/package.json#L4).

Reply:
See 2d54deee